### DISCUSSION TITLE
Can I open the component tree with all nodes collapsed by default?

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#7761] [Help] [answered]
I have a huge screen to manage. When I try to open the designer and look at the component tree, it takes several minutes to expand all nodes there are which are not of interest at the moment. I think this is because of the designer's default behavior of expanding all sub-nodes when you initially view a screen or try to view a parent node.
In short, is there a way to avoid this default behavior? If not, I suggest to add a preference setting to avoid it. I think it would greatly increase productivity for many people including me.

**Comment by melohagan** [comment, marked as answer]
This was discussed recently: https://github.com/Budibase/budibase/discussions/7673

Seems like it would be worth having an expand/collapse button for the component tree. I've raised a feature request here: https://github.com/Budibase/budibase/issues/7763
